dali/dfuXRefLib/dfuxreflib.cpp

```diff
@@ -335,8 +335,9 @@ static unsigned short getDafsPort(const SocketEndpoint &ep,unsigned &numfails,Cr
     }
     else if (numfails>5)
         return 0;
-    if (testDaliServixPresent(ep)) 
-        return ep.port?ep.port:getDaliServixPort();
+    unsigned short nPort = ep.port;
+    if (testDaliServixPresent(ep, &nPort))
+        return nPort?nPort:getDaliServixPort();
```

Comment:
If testDaliServixPresent() succeeds, nPort is always filled (in fact even if doesn't succeed at moment, see prev. comment).
So above setting/overwrititng, then testing of nPort is misleading.
I think above 3 lines is equivalent to:
```
unsigned short nPort;
if (testDaliServixPresent(ep, &nPort))
    return nPort;
```

But would be clearer as:
```
unsigned short nPort = getActiveDaliServixPort(ep);
if (nPort)
    return nPort;
```